bin/lint-docs.js

```diff
@@ -60,8 +60,8 @@ function validateNoTopLevelTitle(file, content) {
 function linkExists(file, link) {
   const isAbsolute = /^\/docs\//.test(link);
   const linkFilePath = isAbsolute ?
-    join(docsPath, link.replace('/docs/', '')) :
-    join(dirname(file), link);
+                       join(docsPath, link.replace("/docs/", "")) :
+                       join(dirname(file), link);
```

Comment:
Aligned with the assignment and doble quotes??? Looks like you're looking for a fight :P
Or was it Prettier?

Reply:
Regarding the quotes, 
it [might have been Prettier](https://prettier.io/docs/en/rationale.html) , but that [can be configured](https://prettier.io/docs/en/options.html#quotes). According to our tslint.json (soon to be replaced with eslint), our favorite flavor is to use single quotes.

Reply:
Reverted.